calico_containers/calico_ctl/node.py

```diff
@@ -62,8 +62,10 @@
 from utils import validate_ip
 from checksystem import check_system
 
-DEFAULT_IPV4_POOL = IPPool("192.168.0.0/16")
-DEFAULT_IPV6_POOL = IPPool("fd80:24e2:f998:72d6::/64")
+DEFAULT_IPV4_POOL_IP = "192.168.0.0/16"
```

Comment:
I guess it's not used in "a lot" of places, but having a global would still be useful.